### ISSUE TITLE
Your file are already deleted by me, Give me $1,000,000 to recover？

### ISSUE THREAD

**Issue by choleraehyq** [bluezooo/bluezooo.github.io#1]
主页内容是预期中的吗？

**Comment by bluezooo** [comment]
Yes, but it works only if you are a student or staff of University of California, Berkeley, and you have cookies for website "box.cs161.org" in your browser.

Please check www.CS161.org  Computer Security  at UC Berkeley and look into the course projects, it is really funny!
